authentication/tests.py

```diff
@@ -5,6 +5,8 @@
 import json
 from django.core import mail
 from rest_framework.test import APIClient
+
```

Comment:
You can remove this empty line

Reply:
Hi! I will remove this is another Pull Request since this one has been merged. Thanks!